docs/animations.md

```diff
@@ -0,0 +1,19 @@
+# Animations
+
+A picture is worth a thousand words. In this spirit, you can use WEBKNOSSOS to create eye-catching animation of your datasets as a video clip. You can use these short movies as part of a presentation, website, for social media or to promote a publication.
+
+// animation video
+
+## Creating an Animation
+
+Creating an animation is easy:
+
+1. Open any dataset or annotation that you want to use for your animation. 
+2. Optionally, load any [pre-computed 3D meshes](./mesh_visualization.md#pre-computed-mesh-generation) for any segments that you wish to highlight.
+3. For larger datasets, use the bounding box tool to create a bounding box around your area of interest. Smaller datasets can be used in their entirety.
+4. From the `Menu` dropdown in navbar at the top of the screen, select "Create Animation".
+5. Configure the animation options as desired, i.e. camera movement or resolution.
+6. Click the `Start animation` button to launch the animation creation.
+
+
+Either periodically check the [background jobs page](./jobs.md) or wait for a an email confirmation to download the animation video file. Creating an animation may take a while, depending on the selected bounding box size and the number of included 3D meshes.
```

Comment:
Add a note about high res only available in paid plans?

Reply:
Done